tests/microos/verify_setup.pm

```diff
@@ -300,6 +304,15 @@ sub run {
     file_tests();
     disk_tests();
 
+    if (get_var('FIRST_BOOT_CONFIG') =~ /combustion/ && !(is_sle_micro('<5.5') || is_leap_micro('<5.5'))) {
```

Comment:
Some image tests on o3 don't have `FIRST_BOOT_CONFIG` set for historical reasons. That needs to be changed or the code here adjusted to `get_var('FIRST_BOOT_CONFIG', 'ignition+combustion')`

Reply:
I would rather update the test variables in this case in o3.

Reply:
Ok. Should probably be done in the medium types as those are currently defining HDD_2=ignition.qcow2. Or maybe that definition should be moved to the test suites...

Reply:
I have checked {jeos,microos}-combustion only, they seem to be fine.

Reply:
Yes, those came later. Plain `microos` doesn't have it.

Reply:
On the second thought, maybe this should be tested in the combustion only test suites. I will replace the regex check with a string one